rllib/utils/replay_buffers/utils.py

```diff
@@ -21,6 +23,38 @@
 logger = logging.getLogger(__name__)
 
 
+def update_priorities_in_episode_replay_buffer(
+    replay_buffer: EpisodeReplayBuffer,
+    config: AlgorithmConfigDict,
```

Comment:
Let's not use `AlgorithmConfigDict` anymore, only ever `AlgorithmConfig` objects.